doc/source/serve/production-guide/kubernetes.md

```diff
@@ -255,6 +255,26 @@ To enable autoscaling in a KubeRay Cluster, you need to set `enableInTreeAutosca
 In most use cases, it is recommended to enable Kubernetes autoscaling to fully utilize the resources in your cluster. If you are using GKE, you can utilize the AutoPilot Kubernetes cluster. For instructions, see [Create an Autopilot Cluster]((https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster)). For EKS, you can enable Kubernetes cluster autoscaling by utilizing the Cluster Autoscaler. For detailed information, see [Cluster Autoscaler on AWS](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md). To understand the relationship between Kubernetes autoscaling and Ray autoscaling, see [Ray Autoscaler with Kubernetes Cluster Autoscaler](kuberay-autoscaler-with-ray-autoscaler).
 :::
 
+## Load balancer
+Set up ingress to expose your Serve application with a load balancer. See [this configuration](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml)
+
+:::{note}
+- Ray Serve runs HTTP proxy on every node, allowing you to use `/-/routes` as the endpoint for node health checks.
+- Ray Serve uses port 8000 as the default HTTP proxy traffic port. You can change the port by setting `http_options` in the Serve config. Learn more details [here](serve-multi-application).
+:::
+
+## Monitoring
+Monitor your Serve application using the Ray Dashboard.
+- Learn more about how to configure and manage Dashboard [here](observability-configure-manage-dashboard).
+- Learn about the Ray Serve Dashboard [here](serve-monitoring).
+- Learn how to set up the Grafana for Dashboard [here](https://ray-project.github.io/kuberay/guidance/prometheus-grafana/)
```

Comment:
It relies on both Prometheus and Grafana. How about this?
```suggestion
- Learn how to set up [Prometheus](prometheus-setup) and [Grafana](grafana) for Dashboard
```

@sihanwang41 Can we send them to this page instead?  https://docs.ray.io/en/latest/cluster/metrics.html#setting-up-your-prometheus-server

Reply:
nice, I wasn't aware of we have this content in the ray doc already.